core/serialize/serialize.cc

```diff
@@ -1693,7 +1693,19 @@ ast::ExpressionPtr SerializerImpl::unpickleExpr(serialize::UnPickler &p, const G
 
 NameRef SerializerImpl::unpickleNameRef(UnPickler &p, const GlobalState &gs) {
```

Comment:
You can delete the `gs` param now.

Reply:
will do in a follow up, i have a stack on this